src/js/components/award/RecipientInfo.jsx

```diff
@@ -0,0 +1,51 @@
+/**
+ * RecipientInfo.jsx
+ * Created by Emily Gullo 01/20/2017
+ **/
+
+import React from 'react';
+import InfoSnippet from './InfoSnippet';
+import RecipientAddress from './RecipientAddress';
+
+const propTypes = {
+    recipient: React.PropTypes.object
+};
+
+export default class RecipientInfo extends React.Component {
+
+    render() {
+        const recipient = this.props.recipient;
+        const recipientDuns = "Not Available";
```

Comment:
Do these mean the API currently doesn't return these two fields? If so, is there a story for it?